src/core/composables/post/usePostDelete.ts

```diff
@@ -1,29 +1,34 @@
-import IPost from '@/core/interfaces/post.interface';
-import { DateHelper } from '@/core/helpers/date-helper';
 import IErrorResponse from '@/core/interfaces/error-response.interface'
 import HttpRequest from '@/core/enums/api.enum';
 import {api} from '@/core/composables/api-client';
-import ErrorCode from '@/core/enums/error-code.enum';
 import errorResponse from '@/core/helpers/error-response';
-import ErrorMessage from '@/core/enums/error-message.enum';
 import messageResponse from '@/core/helpers/message-response';
 import IMessageResponse from '@/core/interfaces/message-response.interface';
+import { Ref,ref } from 'vue'
 
-const usePostDelete = () => {
+const usePostDelete = async (id:number): Promise<{response: Ref<IMessageResponse | undefined>;error: Ref<IErrorResponse | undefined>;}> =>  {
```

Comment:
you can create an interface for this `{response: Ref<IMessageResponse | undefined>;error: Ref<IErrorResponse | undefined>;`

Reply:
also, use kebab case for the filenames.